client/electron/windows/OutlineService/OutlineService/OutlineService.cs

```diff
@@ -895,8 +861,33 @@ private void GetSystemIpv4Gateway(string proxyIp)
                 throw new Exception("no gateway found");
             }
 
+            string gatewayInterfaceIp;
```

Comment:
Perhaps add some comment about what is going on here and why we need this.

Reply:
Removed.